Xamarin.Forms.Platform.GTK/Renderers/BoxViewRenderer.cs

```diff
@@ -67,7 +71,15 @@ private void SetColor(Color color)
             }
         }
 
-        private void SetSize()
+		private void SetCornerRadius(CornerRadius cornerRadius)
+		{
+			if (Element == null || Control == null)
+				return;
+
+			Control.UpdateHasBorderRadius((int)cornerRadius.TopLeft);
```

Comment:
Won't this ignore all of the CornerRadius values except TopLeft?

Reply:
Absolutely!

Reply:
I have made changes to the BoxView control and its Renderer to use all the values. Now all the radius corners are used in GTK.

The most significant change:
```
cr.Arc(x + width - TopRightRadius, y + TopRightRadius, TopRightRadius, Math.PI * 1.5, Math.PI * 2);
cr.Arc(x + width - BottomRightRadius, y + height - BottomRightRadius, BottomRightRadius, 0, Math.PI * .5);
cr.Arc(x + BottomLeftRadius, y + height - BottomLeftRadius, BottomLeftRadius, Math.PI * .5, Math.PI);
cr.Arc(x + TopLeftRadius, y + TopLeftRadius, TopLeftRadius, Math.PI, Math.PI * 1.5);
```